src/stores/MessageStore.js

```diff
@@ -9,17 +9,6 @@ let CHANGE_EVENT = 'change';
 
 let _messages = {};
 
-function _addMessages(rawMessages) {
```

Comment:
Yes @isuruAb, that's right, we shouldn't mix those two, but this was an eslint issue as well, it's an coincidence that it belongs to two kinds of issues.